src/main/java/com/semmle/jira/addon/util/JiraUtils.java

```diff
@@ -138,4 +146,57 @@ public static void createLgtmIssueType() {
       // Issue Type already created
     }
   }
+
+  public static CustomField getConfigKeyCustomField() {
+
+    CustomFieldManager customFieldManager = ComponentAccessor.getCustomFieldManager();
```

Comment:
I think this code should be moved into the UpgradeTask. The UpgradeTask should store the ID of the field in the configuration, so all uses of the field can simply access the field by ID.